dom.bs

```diff
@@ -1322,6 +1373,9 @@ property of the event being dispatched.
     <p>While <var>parent</var> is non-null:</p>
 
     <ol>
+     <li>If the <var>event</var>'s <a for=Event>path</a> <a for=set>contains</a> <var>parent</var>
+     then <a>throw</a> a "{{HierarchyRequestError!!exception}}" {{DOMException}}.
+
```

Comment:
This would be slow. I don't think we can do this in normal case when dispatching events to dom tree. The relevant code in implementations is so hot that extra virtual c++ calls show up, as an example.
We need to ensure that there are now loops when setting up the parents. That is already done for DOM nodes anyhow - DOM tree doesn't contain loops.

Reply:
Would you prefer us to check the chain during setting of the parent? Would that suffice?

Reply:
I think it would, yeah.

Reply:
Okay I think I've resolved this